### PR TITLE
DSA-Day5,6

### PR DESCRIPTION
**Pull request by vaibhav721**

### CODE REVIEW

File: data-structures/linked-list/Detect_loop_in_linked_list.cpp [hunk 1]
```diff
@@ -0,0 +1,66 @@
+#include <iostream>
```
Comment (InclinedScorpio) [inline data-structures/linked-list/Detect_loop_in_linked_list.cpp:1]: please add question url everywhere.

File: data-structures/linked-list/remove_loop_in_list.cpp [hunk 2]
```diff
@@ -0,0 +1,111 @@
+#include <iostream>
```
Comment (InclinedScorpio) [inline data-structures/linked-list/remove_loop_in_list.cpp:1]: please include question url

File: data-structures/linked-list/Detect_loop_in_linked_list.cpp (hunk 1 above)
Comment (vaibhav721) [reply data-structures/linked-list/Detect_loop_in_linked_list.cpp:2]: @InclinedScorpio Added missing question URL's.

Comment (InclinedScorpio) [reply data-structures/linked-list/Detect_loop_in_linked_list.cpp:2]: Thanks Merging

**Pull request merged**